### DISCUSSION TITLE
@UntrustableRus

### DISCUSSION THREAD

**Discussion by hellonearth311** [Prodigy-Hacking/ProdigyMathGameHacking#766] [(D) Developer]
I need you to tell me where to help develop the Mobile WCM. Also, did you manage to make it show an alert?
@UntrustableRus 
Thanks,
hellonearth311.

**Comment by rusprice** [comment]
Unfortunately I'm the sole person working on the mobile WCM, and it wouldn't really be possible to have multiple people working on it. I was unable to make it show an alert because Framed WebKit doesn't support it, but I was able to change a text to prove it works.

**Comment by rusprice** [comment]
https://media.discordapp.net/attachments/819723471234793483/820086670682357810/video0.mp4
